src/abstracts/EventHandler.ts

```diff
@@ -5,7 +5,7 @@ abstract class EventHandler {
 		this.event = event;
 	}
 
-	abstract async handle(...args: any[]): Promise<void>;
+	abstract handle(...args: any[]): Promise<void>;
```

Comment:
What was the reasoning behind removing `async` here?

Reply:
I did this, typescript doesn't allow async to be used with abstract

TS1243: 'async' modifier cannot be used with 'abstract' modifier.

Reply:
In an abstract async does not need to be specified, the promise<void> is already enough